<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SBEM Viewer</string>
  </property>
  <widget class="TileViewer" name="tileviewer"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>39</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_DB"/>
    <addaction name="actionOpen_DB"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="navdock">
   <property name="windowTitle">
    <string>Navigation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Navigation" name="nav"/>
  </widget>
  <widget class="QDockWidget" name="curvedock">
   <property name="windowTitle">
    <string>Curves</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Curves" name="curves"/>
  </widget>
  <widget class="QDockWidget" name="modeDock">
   <property name="windowTitle">
    <string>Mode</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="ModeDlg" name="mode"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_DB">
   <property name="text">
    <string>Open DB...</string>
   </property>
  </action>
  <action name="actionCreate_DB">
   <property name="text">
    <string>Create DB...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TileViewer</class>
   <extends>QWidget</extends>
   <header>TileViewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Navigation</class>
   <extends>QWidget</extends>
   <header>Navigation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Curves</class>
   <extends>QWidget</extends>
   <header>Curves.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModeDlg</class>
   <extends>QWidget</extends>
   <header>ModeDlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
